packages/sync-actions/src/projects-actions.js

```diff
@@ -7,15 +7,26 @@ export const baseActionsList = [
   { action: 'changeLanguages', key: 'languages' },
   { action: 'changeMessagesConfiguration', key: 'messagesConfiguration' },
   { action: 'setShippingRateInputType', key: 'shippingRateInputType' },
+]
+
+export const myBusinessUnitActionsList = [
   {
     action: 'changeMyBusinessUnitStatusOnCreation',
     key: 'myBusinessUnitStatusOnCreation',
+    actionKey: 'status',
   },
   {
     action: 'setMyBusinessUnitAssociateRoleOnCreation',
     key: 'myBusinessUnitAssociateRoleOnCreation',
+    actionKey: 'associateRole',
+  },
+]
+
+export const customerSearchActionsList = [
+  {
+    action: 'changeCustomerSearchStatus',
+    key: 'status',
```

Comment:
That actions, i'm so confused about, it shows on [Representations](https://docs.commercetools.com/api/projects/project#customersearchstatus) but cant map it to the upper lvl [draft](https://docs.commercetools.com/api/projects/project#project).
its either im not getting it or documentation linking isnt right, wdyt?

Reply:
It was indeed missing from the documentation. I opened an issue to add it: https://github.com/commercetools/commercetools-docs/issues/5506